ApiReview/API.Polly.Core/NoDocs/Polly.Hedging/HedgingActionGeneratorArguments.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Polly.Hedging;
+
+public readonly struct HedgingActionGeneratorArguments<TResult>
+{
+    public ResilienceContext PrimaryContext { get; }
+    public ResilienceContext ActionContext { get; }
+    public int AttemptNumber { get; }
+    public Func<ResilienceContext, ValueTask<Outcome<TResult>>> Callback { get; }
```

Comment:
what return type the callback should return in case a void callback is used? object, right?
in another thread the maintainers rejected my suggestion to make voidType or objectType the first class citizen - but we have it exposed here